exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -446,11 +448,23 @@ func (prwe *prwExporter) execute(ctx context.Context, buf *buffer) error {
 		// Reference for different behavior according to status code:
 		// https://github.com/prometheus/prometheus/pull/2552/files#diff-ae8db9d16d8057358e49d694522e7186
 		if resp.StatusCode >= 200 && resp.StatusCode < 300 {
+			prwe.settings.Logger.Info("remote write request successful",
```

Comment:
I feel like this should probably be debug-level logging?  This is essentially request logging

Reply:
Yeah fair point, I was not sure which level we should put this. I was just trying to understand if requests were being made at all.

I'll update this to debug